core/new-gui/src/app/workspace/types/command.interface.ts

```diff
@@ -0,0 +1,62 @@
+import { WorkflowActionService } from "../service/workflow-graph/model/workflow-action.service";
+
+export type commandFuncs =
```

Comment:
If I understand this correctly, t
 - each commandFunc corresponds to a method in WorkflowActionService, except for undoredo.
 - we duplicate changes by passing what is essentially a function name + args to each client

I think we can avoid hardcoding function names for WorkflowActionService by capturing the function name and args with a function decorator.

Here's a simple prototype that prints the function and its args
[patch.txt](https://github.com/Texera/texera/files/8006770/patch.txt)

Reply:
Thanks! I'll maybe use this in the next step.

Reply:
Agreed with the suggestion since most of the method names are directly mapped. However, it also helps to have such an explicit name list so everything is documented in a place.